src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.className = 'wall';
+spider.className = 'spider';
```

Comment:
Why do you need to assign the same class names to the wall and spider? They've already had it. You've found these elements by the classes btw :)
```suggestion
```